prowler/providers/aws/services/neptune/neptune_cluster_multi_az/neptune_cluster_multi_az.metadata.json

```
@@ -1,30 +1,40 @@
+    "Effects/Denial of Service"
+  ],
   "ServiceName": "neptune",
   "SubServiceName": "",
-  "ResourceIdTemplate": "arn:aws:rds:region:account-id:db-cluster",
+  "ResourceIdTemplate": "",
   "Severity": "medium",
   "ResourceType": "AwsRdsDbCluster",
-  "Description": "Check if Neptune Clusters have multi-AZ enabled.",
-  "Risk": "Ensure that your Amazon Neptune graph database clusters are using Multi-AZ deployment configurations to enhance High Availability (HA) through automatic failover to read replicas in the event of a failure such as an Availability Zone (AZ) outage, an internal hardware or network outage, a software failure or in case of planned system maintenance.",
-  "RelatedUrl": "https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html#neptune-9",
+  "Description": "*Amazon Neptune DB clusters* are evaluated for `Multi-AZ` deployment by checking whether the cluster has read-replica instances distributed across multiple Availability Zones.\n\nA failing result indicates the cluster is deployed in a single AZ and lacks read-replicas that enable automatic promotion and cross‑AZ failover.",
+  "Risk": "**Single‑AZ deployment** creates a clear availability single point of failure.\n\n- **Availability**: AZ outage or maintenance can cause prolonged downtime until the primary is rebuilt.\n- **Integrity/Recovery**: Manual recovery increases risk of configuration errors and longer RTOs, impacting operations and compliance.",
+  "RelatedUrl": [
+    "https://answers.microsoft.com/en-us/outlook_com/forum/all/why-is-outlook-blocking-emails-without-my-concent/30cee3c9-99a6-40b7-8061-bd9ca3ef51bc",
+    "https://answers.microsoft.com/en-us/windows/forum/all/installing-pc-health-check/91799fd2-61a2-4878-a7b2-06e106c1988c",
+    "https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html#neptune-9",
+    "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Neptune/multi-az.html#",
+    "https://answers.microsoft.com/en-us/msoffice/forum/all/devices-in-hybrid-azuread-environment-no-longer/92d85af8-30b5-4fa1-b560-3ae6e2c4a1ed"
+  ],
   "Remediation": {
     "Code": {
-      "CLI": "",
-      "NativeIaC": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Neptune/multi-az.html#",
-      "Other": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Neptune/multi-az.html#",
-      "Terraform": "https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/Neptune/multi-az.html#"
+      "CLI": "aws neptune create-db-instance --db-instance-identifier <REPLICA_ID> --db-instance-class <DB_INSTANCE_CLASS> --db-cluster-identifier <CLUSTER_ID> --availability-zone <AZ>",
+      "NativeIaC": "```yaml\nResources:\n  NeptuneReplica:\n    Type: AWS::Neptune::DBInstance\n    Properties:\n      DBInstanceIdentifier: <REPLICA_ID>\n      DBInstanceClass: <DB_INSTANCE_CLASS>\n      DBClusterIdentifier: <CLUSTER_ID>\n      AvailabilityZone: <AZ>\n```",
```

```suggestion
      "NativeIaC": "```yaml\nResources:\n NeptuneCluster:\n Type: AWS::Neptune::DBCluster\n Properties:\n DBClusterIdentifier: \"<DB_CLUSTER_IDENTIFIER>\"\n # Important: Specify multiple Availability Zones\n AvailabilityZones:\n - \"<AZ_1>\"\n - \"<AZ_2>\"\n - \"<AZ_3>\"\n```",
```